src/lib.rs

```diff
@@ -27,8 +41,23 @@ pub enum NEARLedgerError {
     DeviceNotFound,
     /// Error occurred while exchanging with Ledger device
     APDUExchangeError(String),
+    /// Blind signature not supported
+    BlindSignatureNotSupported,
+    /// Blind signature disabled in ledger's app settings
+    BlindSignatureDisabled,
     /// Error with transport
     LedgerHIDError(LedgerHIDError),
+    /// Transaction is too large to be signed
+    BufferOverflow(OnlyBlindSigning),
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub struct OnlyBlindSigning(pub CryptoHash);
+
+impl OnlyBlindSigning {
+    pub fn hash_bytes(bytes: &[u8]) -> Self {
```

Comment:
I don't see any value in exposing this method to be part of the public API. Do you?

Reply:
addressed by [commit](https://github.com/khorolets/near-ledger-rs/pull/4/commits/245d912a5ae3c6dfff2af450334989f48a4e4fbd)